include/picongpu/plugins/binning/Binner.hpp

```diff
@@ -182,11 +195,12 @@ namespace picongpu
                     auto bufferExtent = this->histBuffer->getDeviceBuffer().getDataSpace();
 
                     // allocate this only once?
-                    auto hReducedBuffer = HostBuffer<TDepositedQuantity, 1>(bufferExtent);
+                    // using a unique_ptr here since HostBuffer does not implement move semantics
```

Comment:
The cleaner alternative here might be to implement move semantics for that class